source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -1079,23 +1156,30 @@ namespace aspect
                                Patterns::List(Patterns::Anything()),
                                "This determines what minerals and fabrics or fabric selectors are used used for the LPO/CPO calculation. "
                                "The options are Olivine: Passive, A-fabric, Olivine: B-fabric, Olivine: C-fabric, Olivine: D-fabric, "
-                               "Olivine: E-fabric, Olivine: Karato 2008 or Enstatite. Passive sets all RRSS entries to the maximum. The "
+                               "Olivine: E-fabric, Olivine: Karato 2008 or Enstatite or CPX. Passive sets all RRSS entries to the maximum. The "
                                "Karato 2008 selector selects a fabric based on stress and water content as defined in "
                                "figure 4 of the Karato 2008 review paper (doi: 10.1146/annurev.earth.36.031207.124120).");
 
+            prm.declare_entry ("CPX RRSS", "1,5,5,1.5",
+                               Patterns::List(Patterns::Anything()),
+                               "The default RRSS values for CPX, used in fabric calculations."
+                               "(preliminary results, require further investigations)"
+                               "user can change this in input file under subsection Initial grains"
+                               "by adding a variable CPX RRSS = four entries separated by comma"
+                               "Main slip systems from Bascou_etal., 2002 JSG and "
+                               "Zhang et al., 2006 EPSL and with numerical experiments");
```

Comment:
Phrase this differently:
```suggestion
                               "(preliminary results, pending further investigations)."
                               "This list expects 4 entries separated by commas."
                               "Main slip systems from Bascou et al., 2002 JSG and "
                               "Zhang et al., 2006 EPSL and from numerical experiments");
```